decidim/decidim
# Fix API paths when deploying decidim in folder

#### :tophat: What? Why?
I have a project that is deploying decidim within a path of a domain. And when this is happening, the hardcoded `/api` paths are not working anymore. 

This PR is trying to add path support for Decidim. 

#### Testing
1. Create  anew development application 
2. Edit config/routes.rb and change `mount Decidim::Core::Engine => '/'` to something like ` mount Decidim::Core::Engine => '/participation'`
3. Start the development application & login 
4. Go to Conversations and attempt to start a conversation 
5. See there is no way you can select the users to add to conversation 
6. Apply patch & restart application
7. Try to search for an user, see there are results. 

:hearts: Thank you!
